azure-pipelines.yml

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+jobs:
+    - job: 'tests_and_checks'
+
+      pool:
+          vmImage: 'ubuntu-16.04'
+
+      steps:
+          - task: NodeTool@0
+            inputs:
+                versionSpec: '10.15.3'
+            displayName: use node 10.15.3
+
+          - task: Npm@1
+            inputs:
+                command: custom
+                customCommand: ci
+            displayName: npm ci
+
+          - task: Npm@1
+            inputs:
+                command: custom
+                customCommand: run lint
+            displayName: Check for Lint
+
+          - task: Npm@1
+            inputs:
+                command: custom
+                customCommand: run format-check
+            displayName: check code formatting
+
+          - task: Npm@1
+            displayName: check copyright headers
```

Comment:
Nitpick: Every other task puts displayName at the bottom of the task; can this one be consistent?